helix-common/src/main/java/org/apache/helix/SystemPropertyKeys.java

```diff
@@ -82,4 +82,6 @@
   // System Property Metadata Store Directory Server endpoint key
   public static final String MSDS_SERVER_ENDPOINT_KEY =
       MetadataStoreRoutingConstants.MSDS_SERVER_ENDPOINT_KEY;
+
+  public static final String STATEUPDATEUTIL_ERROR_LOG_ENABLED = "helix.StateUpdateUtil.errorLog.enabled";
```

Comment:
Again, we are disabling log to ZK, not disabling error log (to log file) at all, right?

Reply:
We have a class ZkSystemPropertyKeys to hold this kind of system property. Please put this there.

Reply:
@lei-xia, so let us change the name to " STATEUPDATEUTIL_ERROR_LOG2ZK_ENABLED?

Reply:
@dasahcc, ZkSystemPropertyKeys is for Zookeeper related. This is more to me like logging related.